env/onedplvars.sh

```diff
@@ -16,11 +16,10 @@ ONEDPLROOT=SUBSTITUTE_INSTALL_DIR_HERE
 if [ -n "$1" ]; then
     ONEDPLROOT=$1
 else
-    echo "ERROR: please set oneDPL install dir"
-    return 1;
+    ONEDPLROOT=$(dirname ${BASH_SOURCE})/..
```

Comment:
There are couple things:
1. Minor: set on line 15 looks useless.
2. Major: support of ZSH and KSH should be considered like it is done in [oneTBB](https://github.com/oneapi-src/oneTBB/blob/master/integration/linux/env/vars.sh) or [oneDAL](https://github.com/oneapi-src/oneDAL/blob/master/deploy/local/vars_lnx.sh). It can be handled separately if you prefer.

Reply:
1. Fixed.
2. I'd suggest to address it later since it should be tested with ZSH and KSH. I'm not sure this blocks the PR